examples/kubernetes/README.md

```diff
@@ -35,6 +35,12 @@ They assume Keycloak is used as an authorization server, with properly configure
   A single node Kafka cluster with OAuth 2 authentication with OAuth metrics enabled.
   See [README-metrics.md]() for how to setup this example.
 
+* `kafka-oauth-single-authz-service-accounts.yaml`
+
+  A single node Kafka cluster using [service account tokens](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#service-account-tokens) for authorization and the `simple` authorizer.
+  It requires that the `kube-root-ca.crt` be copied from its ConfigMap to a Secret:
+    
+      kubectl get configmap/kube-root-ca.crt -o=json | jq -r '.data."ca.crt"' | kubectl create secret generic kube-root-ca --from-file=ca.crt=/dev/stdin
```

Comment:
Is this really something what works with every single Kubernetes cluster based on Service accounts only? Or does it really on OpenID provider?

Reply:
The Kubernetes api-server supports [OpenID Connect](https://kubernetes.io/docs/concepts/security/service-accounts/#authenticating-credentials) to validate the credentials which (since version 1.22) get projected into a pod based off of it's service account. You don't need anything outside of a Kubernetes cluster to make use of this authentication.

Reply:
Ok, interesting. I will have to try that.